### PR TITLE
[ranges] Missed kebab-case markings for exposition-only names.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixup for #3559.

### CODE REVIEW

**Comment by johelegp** [comment]
Some of the suggestions in the original PR are missing a fix here.

**Comment by jensmaurer** [comment]
I thought I had them all covered. Anything specific?

**Comment by johelegp** [comment]
Seems like GitHub has problems with big changes. This has happened before. Screenshots to the comments attached.

**Comment by jensmaurer** [comment]
Ok, I hope I got them all this time.

**Comment by jensmaurer** [comment]
Oh, the index question is still open. I'll at fix the spelling, but leave the italics for another day (needs more gymnastics with the indexing macros).

And yes, the question about the inconsistent initialization with the istream iterator stuff is disturbing.

File: source/ranges.tex [hunk 1]
```diff
@@ -5671,14 +5672,14 @@
 Equivalent to:
 \begin{codeblock}
 auto [pcur, pend] = subrange{x.@\exposid{i_}@.@\exposid{parent_}@->@\exposid{pattern_}@};
-auto end = ranges::end(x.@\exposid{i_}@.@\exposid{parent_}@->@\exposid{base_}@);
+auto end = ranges::end(x.@\exposid{i_}@.@\exposid{parent}@->@\exposid{base_}@);
```
Comment (johelegp) [inline source/ranges.tex]:
```suggestion
auto end = ranges::end(x.@\exposid{i_}@.@\exposid{parent_}@->@\exposid{base_}@);
```

File: source/ranges.tex [hunk 2]
```diff
@@ -5009,8 +5009,8 @@
 \pnum
 Let \tcode{\placeholder{inner-range}} be:
```
Comment (johelegp) [inline source/ranges.tex]:
```suggestion
Let \exposid{inner-range} be:
```

File: source/ranges.tex (hunk 1 above)
Comment (jensmaurer) [reply source/ranges.tex]: Fixed.

File: source/ranges.tex (hunk 2 above)
Comment (jensmaurer) [reply source/ranges.tex]: This is really a placeholder for some code, not an exposition-only member. Leaving as-is.

**Pull request merged**